RamilyaNigmatullina/pdp-3
# Add ability to sort posts

# Summary

User should be able to sort posts. Sort options:
* Latest Posts
* Oldest Posts
* Most popular

https://github.com/RamilyaNigmatullina/pdp-3/projects/1#card-19568522

# Test plan

List of steps to manually test introduced functionality:

* Go to Application
* Sort posts
* Check the order has been applied

# Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduces DB fields have indexes and constraints
- routes are RESTful, no useless routes
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

# Deploy notes

None.
